smtp-listeners/src/main/kotlin/no/nav/emottak/nfs/NFSConnector.kt

```diff
@@ -38,7 +40,9 @@ class NFSConnector(jSch: JSch = JSch()) : AutoCloseable {
         sftpChannel.cd("/outbound/cpa")
     }
 
-    fun folder(path: String = "/outbound/cpa"): Vector<ChannelSftp.LsEntry> = sftpChannel.ls(path) as Vector<ChannelSftp.LsEntry>
+    fun folder(path: String = "/outbound/cpa"): Vector<ChannelSftp.LsEntry> =
```

Comment:
default value usage is not appropriate here.

Reply:
Helt enig! Denne virker veldig unødvendig.